docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML using the UI
+title: Configure SAML authentication using the Grafana user interface
+weight: 1150
+---
+
+# Configure SAML authentication using the Grafana user interface
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file. For instructions on how to set up SAML using the Grafana configuration file, refer to [Configure SAML authentication using the configuration file]({{< relref "../saml/" >}}).
+
+The Grafana SAML UI provides the following advantages over configuring SAML in the Grafana configuration file:
```

Comment:
Should we mention as an advantage that by using UI users will get basic validation and feedback about their setup, to ensure the correctness of the config?

Reply:
Good shout, I've added a bulletpoint for it. Let me know what you think.